### PR TITLE
fix: fixing and adding admin ressources

### PR DESCRIPTION
**Pull request by enerisan**

### CODE REVIEW

**Review by yannduhamel — APPROVED** [review]

**Review by angelique888 — DISMISSED** [review]

**Review by AnneDAMIEN — DISMISSED** [review]

File: client/src/main.jsx [hunk 1]
```diff
@@ -9,6 +9,10 @@ import DashboardPage from "./pages/DashboardPage/DashboardPage";
 import App from "./App";
 import AddRecipePage from "./pages/AddRecipePage/AddRecipePage";
 import LogOutPage from "./pages/LogOutPage/LogOutPage";
+import AdminUsersPage from "./pages/DashboardPage/AdminUsersPage/AdminUsersPage";
+import ModifUserPage from "./pages/DashboardPage/ModifUserPage/ModifUserPage";
+import AdminRecipesPage from "./pages/DashboardPage/AdminRecipesPage/AdminRecipesPage";
+/* import ModifRecipePage from "./pages/DashboardPage/ModifRecipePage/ModifRecipePage"; */
```
Comment (ioayoub) [inline client/src/main.jsx:15]: ❌

File: client/src/main.jsx [hunk 2]
```diff
@@ -48,6 +52,28 @@ const router = createBrowserRouter([
...
+        loader: ({ params }) => fetch(`${express}/api/user/${params.id}`),
+      },
+
+      {
+        path: "/admin/recipes",
+        element: <AdminRecipesPage />,
+        loader: () => fetch(`${express}/api/recipe`),
+      },
+
+      /* {
```
Comment (ioayoub) [inline client/src/main.jsx:72]: ❌

**Review by ioayoub — CHANGES_REQUESTED** [review]

File: client/src/main.jsx (hunk 1 above)
Comment (enerisan) [reply client/src/main.jsx]: done

File: client/src/main.jsx (hunk 2 above)
Comment (enerisan) [reply client/src/main.jsx]: done

**Review by Pierre-Voisin — APPROVED** [review]

**Review by yannduhamel — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Review by AnneDAMIEN — APPROVED** [review]

**Pull request merged**